packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/tests/index.test.js

```diff
@@ -100,20 +100,32 @@ describe('ADMIN | Pages | AUDIT LOGS | ListView', () => {
   it('should open a modal when clicked on a table row and close modal when clicked', async () => {
     mockUseQuery.mockReturnValue({
       data: {
-        results: TEST_DATA,
+        results: TEST_PAGE_DATA,
       },
       isLoading: false,
     });
+    render(App);
 
-    const { container } = render(App);
     expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
 
-    const rows = container.querySelector('tbody').querySelectorAll('tr');
-    await user.click(rows[0]);
-    expect(screen.getByRole('dialog')).toBeInTheDocument();
+    mockUseQuery.mockReturnValue({
+      data: TEST_SINGLE_DATA,
+      status: 'success',
+    });
+
+    const auditLogRow = screen.getByText('Create role').closest('tr');
+    await user.click(auditLogRow);
+
+    const modal = screen.getByRole('dialog');
+    expect(modal).toBeInTheDocument();
+
+    expect(within(modal).getByText('Create role')).toBeInTheDocument();
```

Comment:
Could you store the `within` in a variable `const modalContainer = within(modal)` or do you prefer it like this?